internal/help_test.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccVersionHelp(t *testing.T) {
+	stdout, stderr := RequireSuccessfulRun(t, "version", "--help")
+	assert.Empty(t, stderr.String())
+	// We rely on the help message containing the string below in our driver local
```

Comment:
Why can't we just do `databricks version` and check the version of CLI?

Reply:
The web terminal installs the latest version of the CLI every time, irrespective of the DBR version. I wanted something static and specific to the CLI.

Reply:
Hm, I didn't mean to check for specific version, but instead to check that version returned by databricks version is higher than 0.200, I believe that's what's important for test?

Reply:
That's a good point. I wanted to avoid it since it means parsing the text output, but it does seem better than arbitrarily restricting changes to the help output. Closing this PR.